python/cudf/cudf/tests/test_orc.py

```diff
@@ -391,6 +391,57 @@ def test_orc_writer(datadir, tmpdir, reference_file, columns, compression):
     assert_eq(expect, got)
 
 
+@pytest.mark.parametrize("stats_freq", ["NONE", "STRIPE", "ROWGROUP"])
+def test_orc_writer_statistics_frequency(datadir, tmpdir, stats_freq):
+    reference_file = "TestOrcFile.demo-12-zlib.orc"
+    pdf_fname = datadir / reference_file
+    gdf_fname = tmpdir.join("gdf.orc")
+
+    try:
+        orcfile = pa.orc.ORCFile(pdf_fname)
+    except Exception as excpr:
+        if type(excpr).__name__ == "ArrowIOError":
+            pytest.skip(".orc file is not found")
+        else:
+            print(type(excpr).__name__)
+
+    expect = orcfile.read().to_pandas()
+    cudf.from_pandas(expect).to_orc(
+        gdf_fname.strpath, statistics=stats_freq
+    )
+    got = pa.orc.ORCFile(gdf_fname).read().to_pandas()
+
+    assert_eq(expect, got)
+
+
+@pytest.mark.parametrize("stats_freq", ["NONE", "STRIPE", "ROWGROUP"])
+def test_chunked_orc_writer_statistics_frequency(datadir, tmpdir, stats_freq):
+    reference_file = "TestOrcFile.demo-12-zlib.orc"
```

Comment:
Maybe this one can use a smaller file. It takes a very long time as-is and it's fairly redundant with the previous test.
```suggestion
    reference_file = "TestOrcFile.test1.orc"
```

Reply:
This change causes the chunked test to fail. I'm not sure the file path is right...

Reply:
I'll try get the other changes in first.

Reply:
Weird, I used this locally.
file should be at `python/cudf/cudf/tests/data/orc/TestOrcFile.test1.orc`

Reply:
No, it's the read-schema that's the problem. Working on it now.

Reply:
I needed to explicitly state which columns to read from that file. (I'm not familiar with the input, but I suspect there were columns in the file that weren't supported by default. Or something.)

Fixed now. I have used the read-schema from the other tests that use `TestOrcFile.test1.orc`.